makefile.shade

```diff
@@ -574,9 +574,6 @@ var CAN_BUILD_ONECORE = '${Directory.Exists(WIN10_SDK_LIB) && Directory.GetFiles
             {
                 var extraBinaries = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
                 {
-                    "System.Private.DataContractSerialization.dll",
-                    "System.Private.Networking.dll",
-                    "System.Private.ServiceModel.dll",
                     "System.Runtime.Serialization.Primitives.dll"
```

Comment:
Not sure, we added it because we broke some projects when we removed NewtonSoft: #1870